staging/src/k8s.io/apiserver/pkg/endpoints/discovery/aggregated/wrapper_test.go

```diff
@@ -108,6 +108,9 @@ func TestAggregationEnabled(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		if tc.accept == aggregatedV2Beta1JSONAccept || tc.accept == aggregatedV2Beta1ProtoAccept {
+			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, genericfeatures.AggregatedDiscoveryRemoveBetaType, false)
```

Comment:
Your choice on whether you want to update this now or later.  I'll allow it either way.

Reply:
what needs to be updated here? These test disablement of the feature gate.